package.json

```diff
@@ -24,18 +24,18 @@
     "prerender": "ng run donate-frontend:prerender"
   },
   "dependencies": {
-    "@angular/animations": "^10.2.3",
+    "@angular/animations": "^11.0.5",
     "@angular/cdk": "^10.2.7",
-    "@angular/common": "^10.2.3",
-    "@angular/compiler": "^10.2.3",
-    "@angular/core": "^10.2.3",
+    "@angular/common": "^11.0.5",
+    "@angular/compiler": "^11.0.5",
+    "@angular/core": "^11.0.5",
     "@angular/flex-layout": "^10.0.0-beta.32",
-    "@angular/forms": "^10.2.3",
+    "@angular/forms": "^11.0.5",
     "@angular/material": "^10.2.7",
```

Comment:
v11? This should have its own `ng update` and you should get instructions on the Update guide subdomain if you select that the project uses Material.

Reply:
Ahhh I see, I've gone ahead and done this now